config/charts/inferencepool/templates/_helpers.tpl

```diff
@@ -31,3 +31,17 @@ Selector labels
 {{- define "gateway-api-inference-extension.selectorLabels" -}}
 inferencepool: {{ include "gateway-api-inference-extension.name" . }}
 {{- end -}}
+
+{{/*
+Envoy Common labels
```

Comment:
are those really envoy specific? if not, we can call them something like sidecar-proxy?